### PR TITLE
Fixes #13719 - Ostree Branches in a Content View

### PR DESCRIPTION
**Pull request by parthaa**
Shows ostree branch information in a content view per content view
version

### CODE REVIEW

**Comment by parthaa** [comment]
Here are the things that this PR adds -> https://partha.fedorapeople.org/ostree-prs/PR-5874/

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-versions.html [hunk 1]
```diff
@@ -62,6 +62,10 @@
           <div translate ng-if="version.docker_tag_count && version.docker_tag_count > 0">
             {{ version.docker_tag_count }} Docker Tags
           </div>
+          <div translate ng-if="version.ostree_branch_count && version.ostree_branch_count > 0">
+            {{ version.ostree_branch_count }} OSTree Branches
+          </div>
+
```
Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-versions.html]: This blank line looked out of place when I was viewing the file.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/content-views.routes.js [hunk 2]
```diff
@@ -122,7 +122,13 @@ angular.module('Bastion.content-views').config(['$stateProvider', function ($sta
         controller: 'ContentViewVersionContentController',
         templateUrl: 'content-views/versions/views/content-view-version-puppet-modules.html'
     })
-
```
Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/content-views.routes.js]: This blank line seems to be a logical separation in the states. Wondering if it should be kept but placed after your new state.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/content-view-version-content.controller.js [hunk 3]
```diff
@@ -10,7 +10,8 @@
      *   ui-router state.
      */
     function ContentViewVersionContentController($scope, translate, Nutupane, Package, Erratum,
-                                                 PackageGroup, PuppetModule, Repository, ContentViewVersion) {
+                                                 PackageGroup, PuppetModule, OstreeBranch, Repository,
+                                                ContentViewVersion) {
```
Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/content-view-version-content.controller.js]: indentation here is messed up.

File: engines/bastion_katello/test/content-views/versions/content-view-version-content.controller.test.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ describe('Controller: ContentViewsVersionContentController', function() {
             PuppetModule = $injector.get('MockResource').$new();
             ContentViewVersion = $injector.get('MockResource').$new();
             Repository = $injector.get('MockResource').$new();
-
```
Comment (daviddavis) [inline engines/bastion_katello/test/content-views/versions/content-view-version-content.controller.test.js]: Again you've removed a blank line that serves to break up logic.

**Comment by daviddavis** [comment]
Tested and it works. :+1: 

ACK from me pending some small comments and jenkins :green_apple:.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/content-views.routes.js (hunk 2 above)
Comment (parthaa) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/content-views.routes.js]: Newline looked like a mistake to me there :) hence removed it

Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/content-views.routes.js]: It looks like it was on purpose to me?

**Pull request merged**